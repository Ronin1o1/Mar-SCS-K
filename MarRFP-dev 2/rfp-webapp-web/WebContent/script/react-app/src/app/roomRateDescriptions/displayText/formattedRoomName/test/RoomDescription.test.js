import React from "react";
import { mount } from "enzyme";
import toJson from "enzyme-to-json";
import RoomDescription from "../content/RoomDescription";

it("renders without crashing", () => {
    mount( < RoomDescription / > );
});

it("renders correctly", () => {
    const tree = mount( < RoomDescription / > );
    expect(toJson(tree)).toMatchSnapshot();
});