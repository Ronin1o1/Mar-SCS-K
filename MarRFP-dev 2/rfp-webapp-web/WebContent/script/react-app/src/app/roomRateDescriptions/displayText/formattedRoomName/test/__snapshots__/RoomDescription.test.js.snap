// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<RoomDescription>
  <RoomDescriptionContextProvider>
    <form
      id="thisForm"
      name="thisForm"
    >
      <table
        className="section"
      >
        <tbody>
          <tr>
            <td
              className="header"
            >
              Room / Rate Description - Administration - Display Text : Room Description - Channels and Languages
            </td>
          </tr>
          <tr
            className="headerHR"
          >
            <td
              valign="top"
            />
          </tr>
        </tbody>
      </table>
      <table
        className="zero_Height"
      >
        <tbody>
          <tr>
            <td
              className="bold"
            >
              Channel:
            </td>
            <td
              className="field_Name"
            >
              <select
                className="fontSize"
                onChange={[Function]}
              >
                <option
                  value="select"
                >
                  Please select a channel
                </option>
              </select>
            </td>
          </tr>
        </tbody>
      </table>
    </form>
  </RoomDescriptionContextProvider>
</RoomDescription>
`;
